parser/healthchecks.go

```diff
@@ -11,6 +11,10 @@ import (
 	docopt "github.com/docopt/docopt-go"
 )
 
+const (
+	defaultProcType string = "web/cmd"
```

Comment:
can you explain this logic a bit more? I don't see this logic explained anywhere in the docstring. I thought it had to be a single process type.